sal_interfaces/ATDome/ATDome_Events.xml

```diff
@@ -231,6 +280,13 @@
             <Units>second</Units>
             <Count>1</Count>
         </item>
+        <item>
+            <EFDB_Name>dropoutTimer</EFDB_Name>
+            <Description>What is this? How does it differ from doorMoveTimeout?</Description>
```

Comment:
I must be blind but I don't know where you're seeing this....

Reply:
Here is sample full status that Tiago gave me:
```
MAIN AJAR 000
DROP AJAR 000
[OFF] 00
POSN 22.90
RR 000
Emergency Stop Active: 0
Top Comm Link OK: 0
Home Azimuth:  0.00
High Speed (degrees): 5.00
Coast (degrees): 0.50
Tolerance (degrees): 1.00
Encoder Counts per 360: 4018143232
Encoder Counts: 81630126151
Last Azimuth GoTo: 20.00
Azimuth Move Timeout (secs): 120
Rain-Snow enabled: 0
Cloud Sensor enabled: 1
Watchdog Reset Time: 600
Dropout Timer: 5
Reverse Delay: 5
Main Door Encoder Closed: 0
Main Door Encoder Opened: 0
Dropout Encoder Closed: 0
Dropout Encoder Opened: 0
Door Move Timeout (secs): 10
```
Notice that there are two door timeouts: Door Move Timeout and Dropout Timer. The manual, of course, gives no hint about what they mean.

See also the EUI. There is a Dropout Timer near the bottom of the TopConf section, in addition to a Door Move Timeout near the top.